model_tests.rb

```diff
@@ -53,6 +53,15 @@ def test_airterminal_fourpipebeam_osm
     result = sim_test('airterminal_fourpipebeam.osm')
   end
 
+  def test_airterminal_inletsidemixer_rb
+    result = sim_test('airterminal_inletsidemixer.rb')
+  end
+
+  # TODO: To be added in the next official release after: 3.2.1
+  # def test_airterminal_inletsidemixer_osm
+  #   result = sim_test('airterminal_inletsidemixer.osm')
+  # end
+
```

Comment:
Register Rb + TODO for OSM as appropriate (since the fields do not exists prior to 3.2.1+)